src/dev/src/app/tree-view/eager-recursive-tree/eager-recursive-tree.ts

```diff
@@ -41,12 +44,25 @@ export class EagerRecursiveTreeDemo {
     },
   ];
 
-  singleRootSelected: { [key: string]: ClrSelectedState } = {};
-  multiRootSelected: { [key: string]: ClrSelectedState } = {};
+  private buildDefaultSelected(rootMap, selectedMap: SelectedMap = {}) {
```

Comment:
Can we have a type on `rootMap` parameter?

Reply:
Added.